src/shared/2015_wheel_constants.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "shared/robot_constants.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    /**
+     * Creates wheel constants for the 2015 robot
+     *
+     * @return wheel constants for the 2015 robot
+     */
+    WheelConstants_t create2015WheelConstants(void);
```

Comment:
Is there a difference when we refer to something as "Legacy" and "2015"?

Reply:
legacy and 2015 is the same for now (2015 is more specific so I think it's preferred)

Reply:
Yeah, I coined the "legacy" term, but 2015 is better.

Reply:
Does `shared_physics_legacySpeed4ToSpeed3` refer to the same thing? It feels a bit weird since everywhere else except there is referred to as 2015.